src/main/java/com/browserstack/automate/ci/common/tracking/PluginsTracker.java

```diff
@@ -1,24 +1,24 @@
 package com.browserstack.automate.ci.common.tracking;
 
+import com.browserstack.automate.ci.common.BrowserStackBuildWrapperOperations;
```

Comment:
Why do we need this import in `PluginsTracker` class?
If you are using `intellij`, then you can try doing, then you can try `Optimize Imports` feature. Shortcut is probably `cmd + shift + ctrl + L` ... it will help in removing stale imports...

Reply:
Done